### ISSUE TITLE
I can't login to my PC

### ISSUE THREAD

**Issue by DrizztDoUrden** [StrikeOrg/ulterius-server#56]
I am using Windows 10 with MS account. I had PIN enabled but I have also tried without it and it still didn't work for me. Also it shows username as "Система" which is System in Russian.

**Comment by andrewmd5** [comment]
Hi @DrizztDoUrden 

Currently Ulterius only supports password authentication and we have not tested with pins.

However your username issue is something we hope to patch in the coming week. A temporary work around would be to set your primary system language to English  as Ulterius will filter "system" by default then.

**Comment by DrizztDoUrden** [comment]
Thanks for response, @Codeusa
So you mean it is unusable for now unless my system language is English? Can you make it possible to select user to log in as or something like that later?

**Comment by DrizztDoUrden** [comment]
Well, version from git works fine, so looks like it can be closed.

**Comment by andrewmd5** [comment]
Just to check, what version did you have installed? I.e when was the last time you downloaded it from the website. Just curious what suddenly started working.

**Comment by DrizztDoUrden** [comment]
I installed it today. Closing was my bad, I launched it from VS so it was executed from my user and I was able to log in. Already found issue in code, I'll just change user of service via windows tool.

**Comment by andrewmd5** [comment]
Right, when Ulterius runs as a service it has a hard time detecting what the default user is, you can hardcode a username in the source code, in a future update we will just let you input your username.

**Comment by ryvaeus** [comment]
Hi @Codeusa,

I'm having issues logging in as well. I'm using a Microsoft Account on my Windows 10 desktop, and am sure my password is correct. However whether I use my full MS email address, the system-generated 5-character shortname of it, or "MicrosoftAccount\email@domain.com", Ulterius either is stuck on "LOGGING IN..." or "Could not login as `name`".

Is it possible that this is due to my MS account being protected by 2FA?

**Comment by ipangrazio** [comment]
I am having the same issue as @ryvaeus although I do not have 2FA protection. I created a second user account on the PC and it works fine, but the main account that is a Microsoft account does not.
